tigerbeetle.yaml

```diff
@@ -0,0 +1,44 @@
+package:
+  name: tigerbeetle
+  version: 0.13.123
+  epoch: 0
+  description: "The distributed financial accounting database designed for mission critical safety and performance."
+  copyright:
+    - license: Apache-2.0
+
+environment:
+  contents:
+    packages:
+      - zig
+      - busybox
+      - ca-certificates-bundle
+      - build-base
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/tigerbeetledb/tigerbeetle
+      tag: ${{package.version}}
+      expected-commit: 5307525e7f1a04f28c8fcb25fea9f17ce0e2a233
+
+  - runs: |
+      # cpu values from here: https://github.com/tigerbeetle/tigerbeetle/blob/2ab9fd620e53a6d61cb119e48ece4008bedd777d/tools/docker/Dockerfile#L30C79-L30C92
+      if [ ${{build.arch}} == 'aarch64' ]; then
+        zig build -Doptimize=ReleaseSafe -Dcpu=baseline+aes+neon
+      elif [ ${{build.arch}} == 'x86_64' ]; then
+        zig build -Doptimize=ReleaseSafe -Dcpu=x86_64_v3+aes
+      fi
+
+      mkdir -p ${{targets.destdir}}/usr/bin
+      mv zig-out/bin/tigerbeetle ${{targets.destdir}}/usr/bin/
+
+  - uses: strip
+
+update:
+  enabled: true
+  github:
+    identifier: tigerbeetledb/tigerbeetle
+    strip-prefix: v
+    use-tag: true
+    # There are some prereleases with a weird WEEKLY format.
+    tag-filter: 0.
```

Comment:
If we want to only want versions that start with `v` we should use `tag-filter: v`

Reply:
you can test locally by running
```
wolfictl check update  --override-version 0.0.1 tigerbeetle.yaml
```

Reply:
Oh sorry - I was assuming the `strip-prefix: v` above meant this repo had tag prefixes of `v`.  In this case the strip prefix is a NOOP

Reply:
I can clean that up in a followup PR as am fixing up some failing PRs ATM.